### ISSUE TITLE
Component: Micro Footer - Select region modal dialog , there is no label for locale modal

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#3817]
## Detailed description
On Firefox, JAWS

Issue: 
Select region modal dialog , there is no label for locale modal

Expected : Provide appropriate label for the modal dialog using aria-label or aria-labeled

**Comment by kennylam** [comment]
@Praveen-sr The `aria-label` already exists for the locale modal button in micro footer. Are you referring to a different element?
![image](https://user-images.githubusercontent.com/909118/94060836-93914a80-fdb2-11ea-9b03-0459e43df6bb.png)

**Comment by Praveen-sr** [comment]
@kennylam This is the modal we are referring too:

Here the title for the dialog is missing 
![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/e5ebc6a4-8a34-41c6-8880-d9b66afad1d1)
